tests/opentracing_instrumentation/test_traced_function_decorator.py

```diff
@@ -30,6 +31,7 @@
 from opentracing_instrumentation import traced_function
 from opentracing_instrumentation import span_in_stack_context
 
+patch = mock.patch
```

Comment:
I believe that this line isn't really required here since `patch` is used only once.
You can use `@mock.patch` instead.